Docs: Fixed invalid example paths in gitian-building.md

JeremyRand opened the pull request:
The example local paths for "Building fully offline" were missing a forward slash prior to the ".git".  This caused an error when trying to run gbuild, like this:

```
fatal: '/home/user/bitcoin.git' does not appear to be a git repository
fatal: Could not read from remote repository.
```

This PR fixes that.

laanwj commented:
I don't think the `/.git` is necessary at all.

JeremyRand commented:
@laanwj Indeed, you are correct; the `/.git` is optional.  Leaving out the `/` while having the `.git` present does not work, though (which is what the current docs say).  Would you prefer that I modify this PR to remove the `/.git`?

laanwj commented:
Yes, I'd say just leave
```
BTCPATH=/some/root/path/bitcoin
SIGPATH=/some/root/path/bitcoin-detached-sigs
```

JeremyRand committed:
Fixed invalid example paths in gitian-building.md

The example local paths for "Building fully offline" have an extraneous ".git".  This caused an error when trying to run gbuild, like this

fatal: '/home/user/bitcoin.git' does not appear to be a git repository
fatal: Could not read from remote repository.

This commit fixes that.

JeremyRand commented:
@laanwj Done.

fanquake commented:
utACK b06f6a992b4a80c6a9ef2387bfd81e30a407b245

MarcoFalke commented:
utACK b06f6a9
